.github/workflows/deploy.yml

```
@@ -1,136 +1,59 @@
+      run: |
+        # Wait a bit for the service to start
+        sleep 30
+        
+        # Get the public IP from AWS (since it might change)
+        PUBLIC_IP=$(aws ec2 describe-instances \
+          --filters "Name=tag:Name,Values=fixfox-instance-dev" "Name=instance-state-name,Values=running" \
+          --query "Reservations[0].Instances[0].PublicIpAddress" \
+          --output text)
+        
+        echo "Testing API at: http://$PUBLIC_IP:3000/api/api/health"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)